### ISSUE TITLE
Comments

### ISSUE THREAD

**Issue by nvoss12838** [jarichardson/rdrshifter#1]
The comments are still from the lava flow code. Come on man, we are professionals here!

**Comment by jarichardson** [comment]
O yeah maybe I should have removed those.

**Comment by nvoss12838** [comment]
Lolm you around tomorrow?

On Thu, Oct 1, 2015 at 8:56 PM, jarichardson notifications@github.com
wrote:

> O yeah maybe I should have removed those.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/jarichardson/rdrshifter/issues/1#issuecomment-144890452
> .

## 

Nicholas Voss
Graduate Student
School of Geosciences
University of South Florida

**Comment by jarichardson** [comment]
Yeah, I was there this afternoon, but I hear that you had banking issues.

**Comment by nvoss12838** [comment]
Ya we are all good though, got it figured out. Some sketchy Irani banking
conventioneers

On Thu, Oct 1, 2015 at 8:58 PM, jarichardson <notifications@github.com>
wrote:

> Yeah, I was there this afternoon, but I hear that you had banking issues.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jarichardson/rdrshifter/issues/1#issuecomment-144890674>
> .
>



-- 
Nicholas Voss
Graduate Student
School of Geosciences
University of South Florida
